content/downloads.html

```diff
@@ -258,11 +258,11 @@ <h2 id="all-releases">All releases</h2>
 
 <ul>
-  <li>Flink 1.3.2 - 2017-08-05 (<a href="http://www.apache.org/dyn/closer.lua/flink/flink-1.3.2/flink-1.3.2-src.tgz">Source</a>, <a href="http://archive.apache.org/dist/flink/flink-1.3.2/">Binaries</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/">Docs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/java">Javadocs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/scala/index.html">ScalaDocs</a>)</li>
-  <li>Flink 1.3.1 - 2017-06-23 (<a href="http://www.apache.org/dyn/closer.lua/flink/flink-1.3.1/flink-1.3.1-src.tgz">Source</a>, <a href="http://archive.apache.org/dist/flink/flink-1.3.1/">Binaries</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/">Docs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/java">Javadocs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/scala/index.html">ScalaDocs</a>)</li>
-  <li>Flink 1.3.0 - 2017-06-01 (<a href="http://www.apache.org/dyn/closer.lua/flink/flink-1.3.0/flink-1.3.0-src.tgz">Source</a>, <a href="http://archive.apache.org/dist/flink/flink-1.3.0/">Binaries</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/">Docs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/java">Javadocs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/scala/index.html">ScalaDocs</a>)</li>
-  <li>Flink 1.2.1 - 2017-04-26 (<a href="http://www.apache.org/dyn/closer.lua/flink/flink-1.2.1/flink-1.2.1-src.tgz">Source</a>, <a href="http://archive.apache.org/dist/flink/flink-1.2.1/">Binaries</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.2/">Docs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.2/api/java">Javadocs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.2/api/scala/index.html">ScalaDocs</a>)</li>
-  <li>Flink 1.2.0 - 2017-02-06 (<a href="http://www.apache.org/dyn/closer.lua/flink/flink-1.2.0/flink-1.2.0-src.tgz">Source</a>, <a href="http://archive.apache.org/dist/flink/flink-1.2.0/">Binaries</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.2/">Docs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.2/api/java">Javadocs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.2/api/scala/index.html">ScalaDocs</a>)</li>
+  <li>Flink 1.3.2 - 2017-08-05 (<a href="http://archive.apache.org/dist//flink/flink-1.3.2/flink-1.3.2-src.tgz">Source</a>, <a href="http://archive.apache.org/dist/flink/flink-1.3.2/">Binaries</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/">Docs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/java">Javadocs</a>, <a href="http://ci.apache.org/projects/flink/flink-docs-release-1.3/api/scala/index.html">ScalaDocs</a>)</li>
```

Comment:
- there is a double slash now, after `dist` which we could avoid
- also, if we're changing the URLs, why not go with https now?